### ISSUE TITLE
Selecting of a visible event in the event table result in a move of the event table

### ISSUE THREAD

**Issue by PSRCode** [lttng/lttng-scope#65]
1- Do a selection inside the thread view
2- The event table will update to the selection
3- Click on a visible event in the event table
4- Other view update
5- The event table will move even if it is not necessary.

**Comment by PSRCode** [comment]
Not sure if a bug ... in the end ... it was less obvious that it is refreshed to place the selected event in the middle of the view. In a small view it looks weird.

We might want to revisit how we do this.

**Comment by alexmonthy** [comment]
Yeah it's a known limitation of the current implementation. Turns out that it is not straightforward to determine "is the selected event currently in the visible table area?". It should be doable, but would probably require tracking the scrollbar position by ourselves.

For now I just had it recenter on every selection. But it's a very valid comment, if the new selection is already visible then it should not recenter.
